skimage/measure/_regionprops.py

```diff
@@ -349,7 +349,7 @@ def __eq__(self, other):
 
 def regionprops(label_image, intensity_image=None, cache=True):
     """Measure properties of labeled image regions.
-
+    
```

Comment:
In `skimage` we use a convention for blank lines not to contain any whitespaces.
Please, change the commit accordingly.